Resources/Prototypes/Floof/Entities/Mobs/Species/anthromorph.yml

```
@@ -0,0 +1,51 @@
+- type: entity
+  save: false
+  name: Urist McAnthro
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobAnthromorph
+  abstract: true
+  components:
+  - type: HumanoidAppearance
+    species: Anthromorph
+  - type: Icon
+    sprite: Floof/Mobs/Species/Anthromorph/parts.rsi
+    state: full
+  - type: Body
+    prototype: Anthromorph
+  - type: MeleeWeapon
+    hidden: true
+    soundHit:
+      path: /Audio/Weapons/pierce.ogg
+    angle: 30
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Blunt: 2
+        Slash: 3
+  - type: Thirst
+  - type: Carriable # Carrying system from nyanotrasen.
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatHuman
+      amount: 5
+  - type: LanguageKnowledge
+    speaks:
+    - TauCetiBasic
+    - SolCommon
+    understands:
+    - TauCetiBasic
+    - SolCommon
```

```suggestion
  name: Urist McAnthro
  parent: BaseMobSpeciesOrganic
  id: BaseMobAnthromorph
  abstract: true
  components:
  - type: HumanoidAppearance
    species: Anthromorph
  - type: Icon
    sprite: Floof/Mobs/Species/Anthromorph/parts.rsi
    state: full
  - type: Body
    prototype: Anthromorph
  - type: MeleeWeapon
    hidden: true
    soundHit:
      path: /Audio/Weapons/pierce.ogg
    angle: 30
    animation: WeaponArcClaw
    damage:
      types:
        Blunt: 2
        Slash: 3
  - type: Thirst
  - type: Carriable # Carrying system from nyanotrasen.
  - type: Butcherable
    butcheringType: Spike
    spawned:
    - id: FoodMeatHuman
      amount: 5
  - type: LanguageKnowledge
    speaks:
    - TauCetiBasic
    - SolCommon
    understands:
    - TauCetiBasic
    - SolCommon
```
Wait this is just a reskinned human but with the Vulpkanin MeleeWeapon Component. Could you not just put the markings on Human at this point, while adding the requisite marking point selections to base human to permit their use?